tests/integration/test_confirm_refunds.py

```diff
@@ -0,0 +1,233 @@
+import os
+
+from django.core.management import call_command
+import pytest
+import time
+
+from django.test import RequestFactory
+from django.urls import reverse
+from django_scopes import scopes_disabled
+
+from pretix_eth.models import WalletAddress
+
+from pretix.base.models import (
+    Order,
+    OrderPayment,
+    OrderRefund,
+)
+
+
+WEB3_PROVIDER_URI = os.environ.get('WEB3_PROVIDER_URI')
+skip_if_no_web3_provider = pytest.mark.skipif(
```

Comment:
Same comment about this pattern being prone to future mistakes.  See my comment here: https://github.com/esPass/pretix-eth-payment-plugin/pull/144#discussion_r635564883